### ISSUE TITLE
Setting 'sizeInMeters:true' in BillboardGraphics ignores 'rotation' parameter.

### ISSUE THREAD

**Issue by dwhipps** [AnalyticalGraphicsInc/cesium#4120]
I just added sizeInMeters:true and a width and height in meters to the Sandcastle example here:

http://cesiumjs.org/Cesium/Apps/Sandcastle/?src=Hello%20World.html&label=Showcases&gist=928681213846f9f82cd8db0528efa0db

(Be sure to choose "Set Billboard Properties At Creation", and zoom in to the North East USA.

**Comment by hpinkos** [comment]
Thanks @dwhipps!

Here's a trimmed down code example: 
``` javascript
var viewer = new Cesium.Viewer('cesiumContainer');

var entity = viewer.entities.add({
    position : Cesium.Cartesian3.fromDegrees(-75.59777, 40.03883),
    billboard : {
        image : '../images/Cesium_Logo_overlay.png',
        rotation : Cesium.Math.PI_OVER_FOUR,
        
        scale: 30000,
        sizeInMeters : true
    }
});
```

**Comment by dwhipps** [comment]
Any chance this is going to get fixed for the next release?

**Comment by hpinkos** [comment]
Sorry @dwhipps, we haven't had a chance to look into this one yet

**Comment by emackey** [comment]
Looks like @dwhipps tracked down the source of this, it was introduced in 7ce1facd111.

**Comment by emackey** [comment]
Also this is a dup of #3979.

**Comment by emackey** [comment]
Thanks @dwhipps and @bagnell.  This fix will be in the October release, Cesium 1.26.
